packages/web-components/src/components/link-with-icon/link-with-icon.ts

```diff
@@ -50,7 +50,7 @@ class DDSLinkWithIcon extends StableSelectorMixin(BXLink) {
    * @internal
    */
   @property()
-  size = LINK_SIZE.LARGE;
+  size = 'lg' as LINK_SIZE;
```

Comment:
Hmm, having to cast this string seems like a smell. Does the solution from https://dev.to/ivanzm123/dont-use-enums-in-typescript-they-are-very-dangerous-57bh obviate the need for this?

Reply:
Updated to this method.

I had originally gone with the union instead to avoid having to import both an object and a type, but it seems that the type cascades through the inheritance chain on its own just fine, and if the object keeps the same name as the enum, we can make future refactors simpler.